test/control-plane/services/dual-stack/dualstack_test.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we move all of the files that are **not** `_test.go` into a separate directory? We already have a `manifests` directory so it would make sense to have a directory for these input/output test files.

Reply:
The `_test.go` is just one file with few lines on it. The test inputs and outputs are what matter in this directory so I at least would find it odd to move them into a sub-directory.

Reply:
> WDYT about having generate directory with all these in it and just have the foo_test.go along with test inputs and outputs in the test directory?

I think it's kind of fine but then it means we will end up with N input files where N is the k8s version. That's why I was thinking that it would make sense to have them in a separate directory as it is likely they will grow in number.

Reply:
Well, right now every k8s version will need its own test binary and its own directory, so that kind of takes care of that concern. 

We'll need to revisit this question when we can call `NewDaemon` multiple times within one process. There I think organizing them into k8s version directories makes sense.